assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -315,6 +317,9 @@ che.infra.kubernetes.namespace.annotate=true
 # They are used to:
 #  - find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.
 #  - actively label {orch-namespace} with any workspace.
+# NOTE: It is strongly recommended to not change this value because the Che operator relies on these labels and their
```

Comment:
what if the admin wants to add extra labels to the namespaces? Maybe we should re-word e.g.
`It is strongly recommended not to remove the default labels`

Reply:
Please take a look at the improved wording.

Reply:
@metlos I still do not get if the admin can add extra custom labels? e.g. what happens if admin set the following:
`app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspaces-namespace,custom.label=my-custom-label` ? will the operator continue working as expected?

Reply:
yes, it is going to work with extra labels on the namespace. I thought I wouldn't have to spell that out explicitly because that's kind of a standard assumption with labels. Do you think we should make that explicit in the comment?

Reply:
Keeping this to `true` shouldn't do any harm if the admin pre-creates the namespaces. We don't try to update the namespaces if we see the labels and/or annotations already present.